### DISCUSSION TITLE
How to properly deploy Prisma + GraphQL (with multiple clients) on Serverless

### DISCUSSION THREAD

**Discussion by renzoralphpua** [prisma/prisma#5741] [Q&A]
Good Day!

I would just like to ask how would I move forward from my problem.

Currently, i'm trying to deploy my project on serverless, but it's giving me this error:
![image](https://user-images.githubusercontent.com/35352850/108491314-b599e500-72de-11eb-917f-d5d0762ccad0.png)
_I already set SLS_DEBUG=* so that additional information will be shown_

And as I look on to the problem, I think the root cause is, I'm using multiple prisma clients (As discussed on [this comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118)). Since there is a need for my project to connect to multiple database instances.

If I deploy my project with only my default schema, it's working properly, since I import it to using `@prisma/client`, but when I add the additional client and try to deploy it to Serverless, I encounter the error stated above. And I think the primary reason is, I'm importing it using `./prisma/masterlist` (masterlist is the name of the additional client)

I'm not really sure if this problem is related to the webpack configuration, or is solely prisma-related. Any help would really be appreciated. Thank you very much!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
